<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GammaRay::AttachDialog</class>
 <widget class="QWidget" name="GammaRay::AttachDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../ui/resources/ui.qrc">
    <normaloff>:/gammaray/GammaRay-128x128.png</normaloff>:/gammaray/GammaRay-128x128.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QLabel" name="loadingLabel">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/gammaray/GammaRay-48x48.png&quot;/&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;Retrieving the list of processes which use Qt ...&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QWidget" name="listViewPage">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="margin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLineEdit" name="filter"/>
       </item>
       <item>
        <widget class="QTreeView" name="view">
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Pro&amp;be:</string>
           </property>
           <property name="buddy">
            <cstring>probeBox</cstring>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="probeBox"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;Access mode:</string>
           </property>
           <property name="buddy">
            <cstring>accessMode</cstring>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="accessMode">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <item>
            <property name="text">
             <string>Out-of-process, local debugging only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Out-of-process, remote debugging enabled</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>In-process</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <zorder>listViewPage</zorder>
     <zorder>loadingLabel</zorder>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../ui/resources/ui.qrc"/>
 </resources>
 <connections/>
</ui>
